AttributeError: 'DockerClient' object has not attribute 'errors'

https://github.com/PrefectHQ/prefect/actions/runs/7787592712/job/21235116908?pr=11849

```
 ERROR at teardown of test_streams_progress_to_given_stream
...
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: http+docker://localhost/v1.43/containers/80ed26c1fd3fcb4f5a868131b753251ecf2a9b1afb9d492afedc082615282d05/json
...
E       AttributeError: 'DockerClient' object has no attribute 'errors'
```

--- a/tests/fixtures/docker.py
+++ b/tests/fixtures/docker.py
@@ -2,6 +2,7 @@ import sys
 from contextlib import contextmanager
 from typing import Generator
 
+import docker.errors as docker_errors
 import pytest
 from typer.testing import CliRunner
 
@@ -68,7 +69,7 @@ def cleanup_all_new_docker_objects(docker: DockerClient, worker_id: str):
             for image in docker.images.list(filters=filters):
                 for tag in image.tags:
                     docker.images.remove(tag, force=True)
-        except docker.errors.NotFound:
+        except docker_errors.NotFound:
             logger.warning("Failed to clean up Docker objects")
 
 

